packages/ui/react-ui-stack/src/components/Stack.tsx

```diff
@@ -114,19 +78,30 @@ export const Stack = ({
 };
 
 const StackTile: MosaicTileComponent<StackItem, HTMLOListElement> = forwardRef(
-  ({ className, path, isOver, item: { items } }, forwardedRef) => {
+  (
+    { classNames, path, isOver, item: { items, transform, onRemoveSection, onNavigateToSection, SectionContent } },
+    forwardedRef,
+  ) => {
     const { t } = useTranslation(translationKey);
     const { Component, type } = useContainer();
 
     // NOTE: Keep outer padding the same as MarkdownMain.
     return (
-      <List ref={forwardedRef} classNames={mx(className, textBlockWidth, 'm-2 p-2', isOver && dropRing)}>
+      <List ref={forwardedRef} classNames={mx(textBlockWidth, 'm-2 p-2 rounded-lg', isOver && dropRing, classNames)}>
         {items.length > 0 ? (
           <Mosaic.SortableContext items={items} direction='vertical'>
             {items.map((item, index) => (
               <Mosaic.SortableTile
                 key={item.id}
-                item={item}
+                item={{
+                  // Spreading doesn’t always get the id if it is a Proxy(TypedObjectImpl)
+                  id: item.id,
+                  object: item.object,
+                  transform,
+                  onRemoveSection,
+                  onNavigateToSection,
+                  SectionContent,
+                }}
```

Comment:
this feels like the tile needs some other mechanism to pass forward props, because these aren't really the "item" per say

Reply:
I was able to revert this later.